code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -60,6 +60,7 @@ var/global/list/image/splatter_cache=list()
 
 /obj/effect/decal/cleanable/blood/Process()
 	if(world.time > drytime)
+		STOP_PROCESSING(SSobj, src) // Ok no more processing after this
```

Comment:
Why do we have blood processing in the first place...

![image](https://user-images.githubusercontent.com/65828539/136013715-c22dac6c-46ae-42de-b502-438cf4753faa.png)

Reply:
nanako code